src/compiler/codegen/index.js

```diff
@@ -45,7 +45,7 @@ export function generate (
   options: CompilerOptions
 ): CodegenResult {
   const state = new CodegenState(options)
-  const code = ast ? genElement(ast, state) : '_c("div")'
+  const code = ast ? (ast.tag === 'script' ? 'null' : genElement(ast, state) ) : '_c("div")'
```

Comment:
can you add a comment above referencing the original issue?

Reply:
ok, I will add the issue